niworkflows/interfaces/registration.py

```diff
@@ -429,17 +455,27 @@ class EstimateReferenceImage(SimpleInterface):
     output_spec = _EstimateReferenceImageOutputSpec
 
     def _run_interface(self, runtime):
-        ref_name = self.inputs.in_file
+        # Select first EPI file
+        ref_name = self.inputs.in_file[0]
         ref_nii = nb.load(ref_name)
         n_volumes_to_discard = _get_vols_to_discard(ref_nii)
 
         self._results["n_volumes_to_discard"] = n_volumes_to_discard
 
+        if self.inputs.multiecho and (len(self.inputs.in_file) < 2):
+            raise ValueError("Argument 'multiecho' is True but "
+                             "'in_file' has only one element")
```

Comment:
we should keep all input checks either in `_format_arg()`, or at least at the top of the `_run_interface()` to avoid performing unnecessary operations

Reply:
BaseInterfaces do not run `_format_arg()` I think, that's only for command lines I believe